examples/gen_test_data/gen_test_data/requirements.txt

```diff
@@ -0,0 +1,4 @@
+promptflow
```

Comment:
move `req.txt` `req_cloud.txt` `config.ini.example` to be under `examples/gen_test_data`